frontend/src/components/Login.tsx

```diff
@@ -11,12 +12,17 @@ const tailLayout = {
 };
 
 const Login = () => {
+    const history = useHistory();
     const onFinish = async (values: any) => {
         try {
-            if(!values.confirmed) {
-                console.log('Success:', values);
-                const users = await axios.get("http://localhost:3000/users", values)
-                console.log(users)
+            const user = await axios.get(`http://localhost:3000/users`, {
+                params: {
+                    password: values.password,
+                    email: values.username
+                }
+            })
+            if(user.data) {
+                history.push('/home');
             }
         } catch (error) {
```

Comment:
Display error message when username / password is incorrect. You can use this:
https://ant.design/components/message/

Reply:
Okay.